test/e2e/set/scyllacluster/scyllamanager.go

```diff
@@ -267,82 +272,431 @@ var _ = g.Describe("Scylla Manager integration", func() {
+			ContainerName: "scylla-manager",
+			CaptureStdout: true,
+			CaptureStderr: true,
+		})
+		o.Expect(err).NotTo(o.HaveOccurred(), stdout, stderr)
+
+		_, schemaRestoreTaskID, err := managerClient.TaskSplit(ctx, *targetSC.Status.ManagerID, strings.TrimSpace(stdout))
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for schema restore to finish")
+		err = wait.PollUntilContextTimeout(ctx, 5*time.Second, 10*time.Minute, true, func(context.Context) (done bool, err error) {
+			schemaRestoreProgress, err := managerClient.RestoreProgress(ctx, *targetSC.Status.ManagerID, schemaRestoreTaskID.String(), "latest")
+			if err != nil {
+				return false, err
+			}
+
+			return schemaRestoreProgress.Run.Status == managerclient.TaskStatusDone, nil
+		})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Initiating a rolling restart of the target ScyllaCluster")
+		_, err = f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Patch(
+			ctx,
+			targetSC.Name,
+			types.MergePatchType,
+			[]byte(`{"spec": {"forceRedeploymentReason": "schema restored"}}`),
+			metav1.PatchOptions{},
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the target ScyllaCluster to roll out")
+		waitCtx9, waitCtx9Cancel := utils.ContextForRollout(ctx, targetSC)
+		defer waitCtx9Cancel()
+		targetSC, err = controllerhelpers.WaitForScyllaClusterState(waitCtx9, f.ScyllaClient().ScyllaV1().ScyllaClusters(targetSC.Namespace), targetSC.Name, controllerhelpers.WaitForStateOptions{}, utils.IsScyllaClusterRolledOut)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		verifyScyllaCluster(ctx, f.KubeClient(), targetSC)
+		waitForFullQuorum(ctx, f.KubeClient().CoreV1(), targetSC)
+
+		framework.By("Enabling raft in target cluster")
+		_, err = f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Patch(
+			ctx,
+			targetSC.Name,
+			types.JSONPatchType,
+			[]byte(`[{"op":"replace","path":"/spec/scyllaArgs","value":"--consistent-cluster-management=true"}]`),
+			metav1.PatchOptions{},
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the target ScyllaCluster to roll out")
+		waitCtx10, waitCtx10Cancel := utils.ContextForRollout(ctx, targetSC)
+		defer waitCtx10Cancel()
+		targetSC, err = controllerhelpers.WaitForScyllaClusterState(waitCtx10, f.ScyllaClient().ScyllaV1().ScyllaClusters(targetSC.Namespace), targetSC.Name, controllerhelpers.WaitForStateOptions{}, utils.IsScyllaClusterRolledOut)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		verifyScyllaCluster(ctx, f.KubeClient(), targetSC)
+		waitForFullQuorum(ctx, f.KubeClient().CoreV1(), targetSC)
+
+		framework.By("Creating a tables restore task")
+		stdout, stderr, err = utils.ExecWithOptions(f.KubeAdminClient().CoreV1(), utils.ExecOptions{
+			Command:       []string{"sctool", "restore", "--cluster", *targetSC.Status.ManagerID, "--location", objectStorageLocation, "--snapshot-tag", backupProgress.Progress.SnapshotTag, "--restore-tables"},
```

Comment:
multiple lines

Reply:
does it take the safer `=` syntax?

Reply:
done

> does it take the safer = syntax?

sctool uses cobra so most likely